istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -128,20 +135,17 @@ var (
 	reset             = false
 )
 
-func extractConfigDump(kubeClient kube.CLIClient, podName, podNamespace string, eds bool) ([]byte, error) {
-	path := "config_dump"
-	if eds {
-		path += "?include_eds=true"
-	}
+func extractConfigDump(kubeClient kube.CLIClient, podName, podNamespace string, addtionPath string) ([]byte, error) {
+	path := "config_dump" + addtionPath
```

Comment:
Why refactor this? `addtionPath` is less clear to me than the original bool value.

Reply:
Because the path has an additional option, `secretPath`, instead of just edsPath, more possibilities may be added to path in the future. Changing it to `additionPath` can improve scalability, what do you think?

Reply:
I'd prefer a config struct instead of this `additionPath`, which doesn't make sense to me in terms of why/when we need to use it and how we can combine different options.

Reply:
The purpose of this PR is to improve performance by specifying `mask=dynamic_active_secrets`. If `mask=dynamic_active_secrets` is also designed as a bool value like `include_eds`, then the code will have to pass additional parameters and set more judgments. Isn't this more unfriendly?

Reply:
Do you think it would be better if we added a bool parameter like `secret`? Or do you have any other suggestions? Thanks

Reply:
Ok, we can leave it as is for now and improve it later if needed. The feature itself makes sense to me.